src/time-picker/time-display.jsx

```diff
@@ -162,26 +164,27 @@ const TimeDisplay = React.createClass({
     }
 
     return (
-      <div {...other} style={this.prepareStyles(styles.root)}>
-        <div style={this.prepareStyles(styles.box)}>
-          <div style={this.prepareStyles(styles.text)}>
-            <div style={this.prepareStyles(styles.affix)} />
-            <div style={this.prepareStyles(styles.time)}>
+      <div {...other} style={prepareStyles(styles.root)}>
+        <div style={prepareStyles(styles.box)}>
+          <div style={prepareStyles(styles.text)}>
+            <div style={prepareStyles(Object.assign({}, styles.affix))} />
```

Comment:
Interesting question. I think it should... it should send a clone of it's argument before passing it down the transformers.